bundle/config/resources/apps.go

```diff
@@ -0,0 +1,76 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/databricks/cli/libs/log"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/apps"
+)
+
+type App struct {
+	// This represents the id which is the name of the app that can be used
+	// as a reference in other resources. This value is returned by terraform.
+	// This equals to app name and added for symmetry with other resources.
+	ID string `json:"id,omitempty" bundle:"readonly"`
+
+	// SourceCodePath is a required field used by DABs to point databricks app source code
+	// on local disk and use it to point to this source code in the app deployment
+	SourceCodePath string `json:"source_code_path"`
+
+	// Config is an optional field which allows configuring the app following Databricks app configuration format like in app.yml.
+	// When this field is set, DABs read the configuration set in this field and write
+	// it to app.yml in the root of the source code folder in Databricks workspace.
+	// If there’s app.yml defined locally, DABs will raise an error.
+	Config map[string]interface{} `json:"config,omitempty"`
```

Comment:
Because of the linter:

```suggestion
	Config map[string]any `json:"config,omitempty"`
```

Reply:
You can run this to update everything in one shot (after merge of/rebase onto main):

```
gofmt -w -r 'interface{} -> any' .
```

Ref #1959